modules/candidate_profile/test/candidate_profileTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Candidate_profile automated integration tests
+ *
+ * PHP Version 7
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Alexandra Livadas <alexandra.livadas@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+use Facebook\WebDriver\WebDriverBy;
+require_once __DIR__
+    . "/../../../test/integrationtests/LorisIntegrationTestWithCandidate.class.inc";
+/**
+ * CandidateProfileIntegrationTest
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Alexandra Livadas <alexandra.livadas@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class CandidateProfileIntegrationTest extends LorisIntegrationTestWithCandidate
+{
+    /**
+     * Setup the tests and database data
+     *
+     * @return void
+     */
+    function setUp()
+    {
+        parent::setUp();
+        $this->setupConfigSetting("useEDC", "true");
```

Comment:
I'm not sure about this. I included this line here because it was included in other candidate integration tests but I can try to run the tests without it and see if it has an impact.

Reply:
I removed this line (and the entire setup method because it was no longer necessary)